packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -175,9 +260,9 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     // The protocol aims to keep half of the reserve value in gold, thus the reserve ratio
     // is two when the value of gold in the reserve is equal to the total supply of stable tokens.
     if (reserveGoldBalance >= stableTokensValueInGold) {
-      return 0;
+      return FixidityLib.newFixed(0);
     } else {
-      return 5;
+      return FixidityLib.newFixedFraction(5, 1000);
```

Comment:
Consider defining a constant `TOBIN_TAX_NUMERATOR = 5 * 10^22 // 0.005` at the top and returning `FixidityLib.wrap(TOBIN_TAX_NUMERATOR)` here

Reply:
Updated.